pkgs/development/libraries/ocl-icd/default.nix

```diff
@@ -1,26 +1,24 @@
-{lib, stdenv, fetchurl, ruby, opencl-headers, addOpenGLRunpath }:
+{ lib, stdenv, fetchFromGitHub, ruby, autoreconfHook, opencl-headers }:
 
 stdenv.mkDerivation rec {
   pname = "ocl-icd";
-  version = "2.2.10";
+  version = "2.3.1";
 
-  src = fetchurl {
-    url = "https://forge.imag.fr/frs/download.php/810/${pname}-${version}.tar.gz";
-    sha256 = "0f14gpa13sdm0kzqv5yycp4pschbmi6n5fj7wl4ilspzsrqcgqr2";
+  src = fetchFromGitHub {
+    owner = "OCL-dev";
+    repo = "ocl-icd";
+    rev = "1165deedb2ae741cd76e60986f7c2cb5023059d9";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
While we are at it, I would much prefer a simple `let` binding for this rather than a recursive attrs which, in my opinion, should only be used where a `let` binding doesn't solve the problem (we have enough recursion bloating eval times as it is :sweat:).

That said, it's not a major blocker, more of a nitpick.

Reply:
But that has nothing to do with this change. Git tags should be used regardless.

Reply:
Applied both changes.